app/columns.tsx

```diff
@@ -7,6 +7,77 @@ import { ChevronsUpDown, ChevronDown, ChevronRight } from "lucide-react";
 import { PrettyDateRange } from "./dates";
 import { InfoPopover } from "@/components/ui/infopopover";
 
+function getSurfaceDocs(template: string): string | undefined {
```

Comment:
I feel like this wants to be in messageUtils.ts, so that we don't need to go all over the place when adding a surface.  Probably also true for the surfaceTagDocs, now that I think about it.

How about making any array of records indexed by the template identifier and with members including tag, docLink, and color and putting it in that file?